qcodes/instrument_drivers/QuTech/F1d.py

```diff
@@ -20,6 +20,13 @@ class F1d(Instrument):
             individual modules.
 
         module (int): module number as set on the hardware.
+
+    Parameter:
+        enable_remote: set module in remote mode, switched on the front panel then deactivated
```

Comment:
what does this line mean? when I read it, I do not understand: front panel? deactivated "then"?